packages/client/src/features/tools/change-bounds/change-bounds-tool-move-feedback.ts

```diff
@@ -185,10 +185,18 @@ export class FeedbackMoveMouseListener extends DragAwareMouseListener {
         if (!this.tracker.isTracking()) {
             return [];
         }
-        // only reset the move of invalid elements, the others will be handled by the change bounds tool itself
-        this.getElementsToMove(target)
-            .filter(element => this.tool.changeBoundsManager.isValid(element))
-            .forEach(element => this.elementId2startPos.delete(element.id));
+        const elementToMove = this.getElementsToMove(target);
+        if (!this.tool.movementBehavior.allElementsNeedToBeValid) {
+            // only reset the move of invalid elements, the others will be handled by the change bounds tool itself
+            elementToMove
+                .filter(element => this.tool.changeBoundsManager.isValid(element))
+                .forEach(element => this.elementId2startPos.delete(element.id));
+        } else {
+            if (elementToMove.find(element => !this.tool.changeBoundsManager.isValid(element)) === undefined) {
```

Comment:
```suggestion
            if (elementToMove.every(element => this.tool.changeBoundsManager.isValid(element))) {
```

Just a minor nitpick but I think it makes the code more readable.